avidemux_fr.ts

```diff
@@ -14099,7 +14099,7 @@ S&apos;il s&apos;agit d&apos;un profil système, il sera recréé la prochaine f
     <message>
         <location line="+1"/>
         <source>_Number of B frames:</source>
-        <translation type="unfinished"></translation>
+        <translation>-Nombre d'image B</translation>
```

Comment:
There is a small typo here ("-Nombre" should be "_Nombre"). An underscore is an accelerator to navigate directly to the corresponding field.

Reply:
You right ! It is a mistake. I create a new pull request with the changes.